toolium/utils/dataset.py

```diff
@@ -66,6 +66,7 @@ def replace_param(param, language='es', infer_param_type=True):
         [TIMESTAMP] Generates a timestamp from the current time
         [DATETIME] Generates a datetime from the current time
         [NOW] Similar to DATETIME without milliseconds; the format depends on the language
+        [NOW('%Y-%m-%dT%H:%M:%SZ')] Same as NOW but using an specific ISO-8601 extended local date format
```

Comment:
It looks that you have finally decided to go for a generic approach. rather than the more rigid but simpler `NOW(ISO)` alternative. A couple of comments then:

- Given the format is open, the description should not make reference to the ISO-8601 standard (which is not necessarily the case), but to the `strftime()` allowed formats at most (as you do in the `_replace_param_date()` docstring).
- If you foresee `NOW('%Y-%m-%dT%H:%M:%SZ')` is gonna be the format used most of the time within this pattern, I think it makes sense to create an alias for it, so that, for example, the previously mentioned `NOW(ISO)` equals that expression. Otherwise it is gonna be quite ugly and more prone to errors.

Reply:
1) you're absolutely right. Let me change it.
2) There are some troubles with this:
First, I don't know what format is going to be used the most because that depends on the implementations. Some providers use that format and some others require including milliseconds, or replacing the 'T' by a blank, and so on. I know the format used by a certain API implementation I'm testing but I cannot assure that's is or will be the most used one
On the other hand, 'ISO' cannot be linked to an specific format. ISO-8901 defines a list of datetime formats and their requirements (they must be lexicographically sorted, etc..) but it is not fitted to an specific format. In the case of creating aliases, that word is not maybe the best. Besides, the code using toolium could have an alias to abstract of the dateformat used here. If you find a suitable better way to hadle this, I'm all ears open

Reply:
Alright, I was just thinking of ways to make the expression shorter and more readable. It is true that you can always resort to your project settings and do something like `[NOW([CONF:time_format])]`. Maybe that is enough.

That made me think of something else: the apostrophes. Are they needed for any reason? If not, I would remove them, because we do not use them in any tag. I would specify `NOW(<format>)` (e.g. `NOW(%Y-%m-%dT%H:%M:%SZ)`), without them.

Reply:
No reason for quotes. Removed as suggested